2024 Conference Code of conduct (#450)
nordic-rse/nordic-rse.github.io: content/events/2024-in-person-conference/CoC_info_for_organizers.md

+++
title = "Code of Conduct Information for Organizers"
+++

# Code of Conduct Information for Organizers

> This code of conduct is copied and adapted from the example policy at the
> [Geek Feminism wiki](https://geekfeminism.wikia.org/wiki/Conference_anti-harassment/Policy),
> created by the Ada Initiative and other volunteers (CC-0 license).

> > The procedure for reporting harassment has been adopted from the Ada Initiative's guide titled

Reviewer: ```suggestion
> The procedure for reporting harassment has been adapted from the Ada Initiative's guide titled
```

> ["workshop anti-harassment/Responding to Reports"](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports).

Please read this before organizing a Nordic-RSE event. This page describes how we
enforce security and the code of coduct in a Nordic-RSE event.

### Warnings

Any member of conference organizer or code of conduct committee member can issue

> a verbal warning to a participant that their behavior violates the conference's

Reviewer: ```suggestion
a verbal warning to a participant whose behavior violates the conference's
```
